.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -1,102 +1,102 @@
-name: Code for Africa | Deploy | REVIEW APP
+# name: Code for Africa | Deploy | REVIEW APP
```

Comment:
Are we doing away with this?

Reply:
Until we figure out how to make review apps work with all the apps, I'm inclined to say yes.